frontend/server/src/main/java/org/pytorch/serve/wlm/Model.java

```diff
@@ -137,6 +149,23 @@ public void setMaxBatchDelay(int maxBatchDelay) {
         this.maxBatchDelay = maxBatchDelay;
     }
 
+    public String getTorchAPIType() {
+        return torchAPIType;
+    }
+
+    public void setTorchAPIType(String torchAPIType) {
+        if (torchAPIType != null) {
+            this.torchAPIType = torchAPIType;
+        } else {
+            String tmpTorchAPIType = this.modelArchive.getManifest().getModel().getTorchAPIType();
+            if (tmpTorchAPIType != null) {
+                this.torchAPIType = tmpTorchAPIType;
+            } else {
+                this.torchAPIType = ConfigManager.getInstance().getTsDefaultTorchAPIType();
```

Comment:
For customer upgrading existing deployments to new release, the default should be set to Python even when there is no value specified in the config file. Will this handle for upgrade scenarios?